.github/workflows/release.yml

```diff
@@ -36,25 +34,37 @@ jobs:
           node-version: 18
 
       - run: npm install
-
       - run: npm run build
-
       - run: npm run test
         if: ${{ inputs.skip-test == false }}
+      - name: Upload release artifacts
+        id: upload-release
+        uses: actions/upload-artifact@v4
+        with:
+          name: release
+          path: lib
 
       - name: Configure AWS Credentials
-        uses: aws-actions/configure-aws-credentials@v2
+        uses: aws-actions/configure-aws-credentials@v4
         with:
           role-to-assume: ${{ secrets.AWS_CODEARTIFACT_ROLE }}
           aws-region: us-west-2
-      - name: Login and configure codeartifact
+      - name: Run CodeBuild
+        uses: aws-actions/aws-codebuild-run-build@v1
+        with:
+          project-name: Importer
+          disable-source-override: true
+          hide-cloudwatch-logs: true
+          env-vars-for-codebuild: |
+            GITHUB_ARTIFACT_ID,
+            GITHUB_REPOSITORY_NAME,
+            GITHUB_COMMIT_MESSAGE,
+            GITHUB_BRANCH_NAME,
+            GITHUB_TOKEN
         env:
-          CODE_ARTIFACT_REPO: ${{ startsWith(github.ref_name, 'dev-v3-') && format('AwsUI-Artifacts-{0}', github.ref_name) || 'github-artifacts' }}
-        run: |
-          echo Logging into repository $CODE_ARTIFACT_REPO
-          aws codeartifact login --tool npm --repository $CODE_ARTIFACT_REPO --domain awsui --domain-owner ${{ secrets.AWS_ACCOUNT_ID }} --region us-west-2 --namespace @cloudscape-design
+          GITHUB_ARTIFACT_ID: ${{ steps.upload-release.outputs.artifact-id }}
+          GITHUB_REPOSITORY_NAME: ${{ github.event.repository.name }}
+          GITHUB_COMMIT_MESSAGE: ${{ github.event.commits[0].message }}
+          GITHUB_BRANCH_NAME: ${{ github.head_ref || github.ref_name }}
```

Comment:
Why is there branching?

Reply:
Can remove since I didn't end up using it anyway. The values are different depending on the context. If you're in a PR then the branch name is in `head_ref` but when pushing to the branch its in `ref_name`..

Reply:
Yes, that's cleaner, thanks